File: 이종호/9월4주차/12915/12915.py
def solution(strings, n):
    answer = []
    n_keyword = [] #문자열의 n번째 단어를 저장해놓을 리스트
    strings.sort() #추가!
    
    for i in strings:
        keyword = i[n]
        n_keyword.append(keyword)
    
    n_keyword.sort()
    
    for i in n_keyword: # n번째 알파벳 기준으로
        for j in strings:
            if i == j[n] and j not in answer:
                answer.append(j)
    
    return answer

File: 구은성/11월/1123/12915.java
import java.util.*;
class Solution {
    public String[] solution(String[] strings, int n) {
        String[] answer = new String[strings.length];
        
        for(int i = 0; i < strings.length; i++) {
            strings[i] = strings[i].charAt(n) + strings[i];
        }
        
        List<String> list = Arrays.asList(strings);
        
        list.sort(Comparator.naturalOrder());
        
        for(int i = 0; i < list.size(); i++) {
            answer[i] = list.get(i).substring(1, list.get(i).length());
        }
        
        return answer;
    }
}